### ISSUE TITLE
Support for listings that are "on sale"

### ISSUE THREAD

**Issue by sharpobject** [davidsawyer/airbnb-price-per-night-correcter#7]
Hello,

This currently doesn't support listings that are "on sale." These look like this: https://i.imgur.com/uYbHrwa.png

The relevant bit of the dom is like this:
```
<div class="_36rlri"><span class="_krjbj">Previous price:</span><div class="_36rlri" style="margin-right: 2px;"><span class="_h6avcp2"><span class="_32pxry">$175</span></span></div> <span class="_krjbj">Discounted price:</span><span class="_1p0spma2"><span>$150</span></span> <span class="_j1kt73"><span class="_krjbj"><div class="_1p3joamp airbnb-pernight-price-correcter-has-modified-this">per night, including fees</div><div id="airbnb-pernight-price-correcter-checkmark" class="{&quot;airbnb-pernight-price-correcter-has-modified-this&quot;}"></div></span><span aria-hidden="true" class="_so3dpm2">/ night</span></span></div>
```

**Comment by davidsawyer** [comment]
Yes, thank you! I only noticed this new (?) concept a few days ago. Not sure how long it's existed for. I'll try to tackle this as soon as I get some free time.

**Comment by davidsawyer** [comment]
Hey @sharpobject and @dmitry, this is now addressed as of version 0.5.0. Enjoy!
